### ISSUE TITLE
Consider not using the wheel event for detecting scroll-position

### ISSUE THREAD

**Issue by razh** [ten1seven/what-input#76]
An unfortunate side-effect of adding a `wheel` event listener is that it triggers undesirable scrolling behavior for `input[type="number"]` elements in Chrome and Safari on macOS:

```html
<label for="exampleNumberInput">Example number input</label>
<input type="number" class="form-control" id="exampleNumberInput" name="exampleNumberInput">
```

![2018-02-21-number-input-wheel-events](https://user-images.githubusercontent.com/240770/36465401-d9e702e4-16a2-11e8-8245-3244833973c9.gif)

This only occurs when a `wheel` event listener is added and it occurs regardless of where in the document it's added. For example, these `input[type="number"]` events don't have this behavior:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number

Currently, the prevailing solution is to `blur` the input on the `mousewheel` event, but I would like to keep focus on the element:

https://stackoverflow.com/questions/9712295/disable-scrolling-on-input-type-number

I don't know the full extent of how this behavior affects Windows and Linux.

**Comment by ten1seven** [comment]
Thanks for the detailed ticket @razh! I'll take a look into this.

**Comment by razh** [comment]
@ten1seven No worries, we decided to go another route on this that appears to work. I'm aware of at least Firefox on Windows and Linux exhibiting this behavior even without the `.addEventListener('wheel')`.

For future reference, we used [jquery-mousewheel](https://github.com/jquery/jquery-mousewheel) to normalize scroll distances and added it to `scrollTop` in the `mousewheel` event listener instead of blurring.

Feel free to close this!
